mautic/mautic
# Add default parameters to public method

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
In our app we process parameters in different way like in Mautic. Then we need process $defaultParameters static variable in public method. This PR added it.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Just code review. Break nothing




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11259"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
